stylus/cozy-ui/defaults.styl

```diff
@@ -42,6 +23,34 @@ body
     @extend $font-labor
 
 
+// UTILS
+@import '../ui-base/mixins'
+:global
+    .coz-hidden
+        @extend $hidden
+
+    .coz-desktop
+        @media (max-width: (1023/basefont)rem)
+            display none !important
```

Comment:
Why the `!important`? Did you get any trouble without it? Or did you see some trouble in some use cases I didn't think of?